homeassistant/components/roborock/strings.json

```diff
@@ -108,6 +108,14 @@
         "name": "Status indicator light"
       }
     },
+    "time": {
+      "dnd_start_time": {
+        "name": "DnD begin"
```

Comment:
The only downside is that it gets rather long and you can't really read it if it is the full thing
![image](https://github.com/home-assistant/core/assets/20257911/8613b8bb-5dac-43fd-944e-9d4f299c5c11)

Reply:
We should set the name to what we think it should be, then address the UI in some separate way (e.g. assume it may cut off even if its "DND Begin" even its just slightly easier to read...).  For example, Mop intensity is a wide select bar and it includes the name of it in the select bar and maybe that needs to happen here too.

Reply:
Is there a way to include the name inside the selection for time? Or would that take changing the Time platform itself on the frontend?

Reply:
I'm suggesting a frontend change (or a bug filed?). I have only a little experience with frontend but it seems possible....

Reply:
Created an issue here: https://github.com/home-assistant/frontend/issues/16937

For now I have changed this to be do not disturb start and do not disturb end.

The icons fortunately display a little bit of information to make it clearer even if it is cutoff. I wanted to do begin do not disturb and end do not disturb, but since the entitles are sorted in alphabetical order on the dashboard, that made them very far away onthte dashboard from each other and even more confusing